projects/arlas-toolkit/src/lib/components/top-menu/top-menu.component.ts

```diff
@@ -75,4 +109,35 @@ export class TopMenuComponent implements OnInit {
   public getUserInfos() {
     this.dialog.open(UserInfosComponent);
   }
+
+  public getInitials(name) {
```

Comment:
Shouldn't this be a method in an utils file ? I think the exact same method is used in multiple places in both the toolkit and the WUI.

Reply:
The idea here, is to add the top menu in all fronts, so we won't need to re-implement it